web/admin/src/request/types.ts

```diff
@@ -1390,7 +1394,8 @@ export interface V1RssScrapeResp {
 }
 
 export interface V1ScrapeReq {
-  url?: string;
+  kb_id: string;
+  url: string;
```

Comment:
在 `V1ScrapeReq` 接口中，`url` 字段从可选变为了必填。请确认这是否符合后端接口的实际要求，如果后端接口仍然接受可选字段，则此处修改可能会导致不必要的限制。

```suggestion
// 如果后端接口支持可选字段，请保持一致性
url?: string;